dashboard/db/schema.rb

```diff
@@ -1710,14 +1710,12 @@
     t.string "address_line3", limit: 30, comment: "Location address, street 3"
     t.decimal "latitude", precision: 8, scale: 6, comment: "Location latitude"
     t.decimal "longitude", precision: 9, scale: 6, comment: "Location longitude"
-    t.string "state_school_id"
     t.string "school_category"
     t.string "last_known_school_year_open", limit: 9
     t.index ["id"], name: "index_schools_on_id", unique: true
     t.index ["last_known_school_year_open"], name: "index_schools_on_last_known_school_year_open"
     t.index ["name", "city"], name: "index_schools_on_name_and_city", type: :fulltext
     t.index ["school_district_id"], name: "index_schools_on_school_district_id"
-    t.index ["state_school_id"], name: "index_schools_on_state_school_id", unique: true
```

Comment:
Should I update the schema here or in the PR with the actual database changes?

Reply:
please wait, this will also be taken care of by the migration